docs/pages/starter-projects.md

```diff
@@ -67,7 +67,7 @@ npm install
 npm start
 ```
 
-Once compiled, you project is viewable at: <a class="button primary" href="http://localhost:8000" target="_blank">http://localhost:8000</a>
+Once compiled, you project is viewable at: <a class="button primary" href="https://localhost:8000" target="_blank">//localhost:8000</a>
```

Comment:
There is normally no https on localhost except people use mkcert (with ddev) and so on.

Reply:
Also please add the protocol here (we should keep it with http://).

Reply:
Yep. Noticed that as well. just fixed that one. Will push soon.